CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+v0.3.0-beta1
+============= 
+
+* using libdaemon to daemonize the ProxySQL process
+* added angel process that is able to monitor and restart ProxySQL
+* added separate error log
+* using MariaDB client library + libevent for connecting and querying backend MySQL servers
+* added a per-user:
+  * max_connection value that limits the number of connections from the same user
+  * transaction_persistent flag that keeps transactions within the same hostgroup for a certain user 
+* add debug build target (make debug will now build a binary with vs. make)
+* introduce support for packets larger than 16 MB (the maximal single packet size from MySQL binary protocol)
+* handling in-memory stats database locking errors gracefully by retrying (this leads to less errors when gathering stats under load)
+* added SHOW PROCESSLIST command for admin interface that aggregates running queries from all backends
```

Comment:
command SHOW PROCESSLIST is not implemented yet : processlist is implemented through a table. Should be easy to create such new command as an alias for SELECT * FROM stats_mysql_processlist